packages/docs/pages/Table/TablePage.tsx

```diff
@@ -0,0 +1,166 @@
+import { H0, H1, H2, Table, Text } from '@bigcommerce/big-design';
+import { TableItem } from '@bigcommerce/big-design/dist/src/components/Table/types';
+import React from 'react';
+
+import { CodePreview, Collapsible } from '../../components';
+import { MarginPropTable, TableColumnsPropTable, TablePropTable, TableSelectablePropTable } from '../../PropTables';
+
+interface Item extends TableItem {
+  sku: string;
+  name: string;
+  stock: number;
+}
+
+const data: Item[] = [
+  { sku: 'SM13', name: '[Sample] Smith Journal 13', stock: 25 },
+  { sku: 'DPB', name: '[Sample] Dustpan & Brush', stock: 34 },
+  { sku: 'OFSUC', name: '[Sample] Utility Caddy', stock: 45 },
+  { sku: 'CLC', name: '[Sample] Canvas Laundry Cart', stock: 2 },
+  { sku: 'CGLD', name: '[Sample] Laundry Detergent', stock: 29 },
+  { sku: 'TWB', name: '[Sample] Tiered Wire Basket', stock: 119 },
+  { sku: 'OCG', name: '[Sample] Oak Cheese Grater', stock: 34 },
+  { sku: 'SLLPJ', name: '[Sample] 1 L Le Parfait Jar', stock: 7 },
+  { sku: 'CC3C', name: '[Sample] Chemex Coffeemaker 3 cup', stock: 49 },
+  { sku: 'ABS', name: '[Sample] Able Brewing System', stock: 225 },
+  { sku: 'OTS', name: '[Sample] Orbit Terrarium - Small', stock: 89 },
+  { sku: 'OTL', name: '[Sample] Orbit Terrarium - Large', stock: 109 },
+  { sku: 'SLCTBS', name: '[Sample] Fog Linen Chambray Towel - Beige Stripe with some fondu of some sort', stock: 49 },
+];
+
+const columns = [
+  { header: 'Sku', render: ({ sku }) => sku },
+  { header: 'Name', render: ({ name }) => name },
+  { header: 'Stock', render: ({ stock }) => stock },
+];
+
+export default () => {
+  return (
+    <>
+      <H0>Table</H0>
+
+      <CodePreview>
+        {/* jsx-to-string:start */}
+        <Table
+          columns={[
+            { header: 'Sku', render: ({ sku }) => sku },
+            { header: 'Name', render: ({ name }) => name },
+            { header: 'Stock', render: ({ stock }) => stock },
+          ]}
+          items={[
+            { sku: 'SM13', name: '[Sample] Smith Journal 13', stock: 25 },
+            { sku: 'DPB', name: '[Sample] Dustpan & Brush', stock: 34 },
+            { sku: 'OFSUC', name: '[Sample] Utility Caddy', stock: 45 },
+            { sku: 'CLC', name: '[Sample] Canvas Laundry Cart', stock: 2 },
+            { sku: 'CGLD', name: '[Sample] Laundry Detergent', stock: 29 },
+          ]}
+        />
+        {/* jsx-to-string:end */}
+      </CodePreview>
+
+      <H1>API</H1>
+
+      <H2>Table</H2>
+
+      <TablePropTable />
+
+      <H2 id="table-columns-prop-table">Table[Columns]</H2>
```

Comment:
Just thinking maybe `PropTable` can accept `title` param to include? So that if we decide to change how the title looks like, we do not need to go to all the component to change them

Reply:
Good call, I initially just made it compatible with the new `<Table>` component but this is solid advice.

I addressed it in a new commit, added a `title` and `collapsible` prop since when we render "Inherited Props" tables we show the title in a different way and collapse the table by default.